src/applications/static-pages/sass/modules/_m-education-wizard.scss

```diff
@@ -53,6 +53,19 @@
     margin-left: -4rem;
   }
 }
+.circle-bullet {
+  list-style-type: circle;
+}
+.ul-styling {
+  margin: -2px;
+}
+li.li-styling{
+  margin-bottom: 0.2em;
+}
+.li-styling2{
+  margin-bottom: -0.5em;
+}
+
```

Comment:
I think all of these margin definitions could be replaced by using the [margin utilities](https://design.va.gov/utilities/margins) from the design system, to save on some CSS.

For example, I think you could the `li.li-styling` rule then apply `vads-u-margin-bottom--0p5` to the elements instead.

Reply:
`li.li-styling` is 0.2em which is about 3px, so i wouldn't be able to use any of those classes. But i did update `li-styling2` to a `vads-u-margin-bottom--neg1`

Reply:
Ah okay, I didn't if you were glued to an exact value for some of those or not